### ISSUE TITLE
[a11y]: "Undersized target "button" does not have sufficient spacing of 12 CSS pixels from another target "button"

### ISSUE THREAD

**Issue by cheenaguptaibm** [carbon-design-system/carbon#18010]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.69.0

### React version

18.3.1

### Automated testing tool and ruleset

IBM Equal Accessibility Checker

### Description

I am getting the following accessibility issue "Undersized target "button" does not have sufficient spacing of 12 CSS pixels from another target "button" while using tab component from carbon/react library.

### Reproduction/example

https://stackblitz.com/edit/github-jnx3gw?file=src%2FApp.jsx

### Steps to reproduce

Run IBM Equal Accessibility Checker on the component and we get following violation
"Undersized target "button" does not have sufficient spacing of 12 CSS pixels from another target "button"

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by guidari** [comment]
Hey @cheenaguptaibm 
This is a false positive that will be fixed with the next ruleset release of accessibility-checker
https://github.com/carbon-design-system/carbon/issues/17353
